thanos/charts/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: 2025 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Please remove those headers. They will prevent Helm from templating if the file is empty.